Formula/s/sq.rb

```diff
@@ -20,7 +20,7 @@ class Sq < Formula
 
   uses_from_macos "sqlite" => :test
 
-  conflicts_with "squirrel", because: "both install `sq` binaries"
```

Comment:
these conflicts can be done as followup

Reply:
Removed these from squirrel and sq. Should `conflicts_with` remain in in this formula?